common/workunit/workunit.cpp

```diff
@@ -14137,23 +14137,31 @@ KeepK8sJobs translateKeepJobs(const char *keepJob)
     return KeepK8sJobs::none;
 }
 
+// NB: will fire an exception if command fails (returns non-zero exit code)
+static void runKubectlCommand(StringBuffer *output, const char *title, const char *cmd, const char *input, bool logOutput, bool logErrors)
+{
+    StringBuffer _output, error;
+    if (!output)
+        output = &_output;
+    bool ret = runExternalCommand(title, *output, error, cmd, input);
+    if (logOutput && output->length())
+        DBGLOG("%s: ret=%u, stdout=%s", cmd, ret, output->trimRight().str());
```

Comment:
If output contains \n this should really be split into multiple DBGLOG calls (I think there's a function to do so somewhere but I can't remember where/what it is called)

Reply:
I've looked, but can't find such a function.
I can add one to jlib (I think you'd also want to use it https://github.com/hpcc-systems/HPCC-Platform/pull/15130 ?)